packages/react-components/review/stable/react-components.api.md

```diff
@@ -122,18 +122,23 @@ export interface CameraButtonStyles extends ControlBarButtonStyles {
     menuStyles?: Partial<CameraButtonContextualMenuStyles>;
 }
 
+// Warning: (ae-internal-missing-underscore) The name "CaptionInfo" should be prefixed with an underscore because the declaration is marked as @internal
+//
 // @internal
-export const _Caption: (props: _CaptionProps) => JSX.Element;
+export type CaptionInfo = {
+    displayName: string;
+    caption: string;
```

Comment:
maybe call it captionText or something? Not to confuse this with caption object instance